assets/mobile-dock.js

```
@@ -0,0 +1,61 @@
+class MobileDock extends HTMLElement {
+  constructor() {
+    super();
+
+    new theme.initWhenVisible(this.init.bind(this));
+  }
+
+  get section() {
+    return this._section = this._section || this.closest('.mobile-dock-section');
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)